src/lib/api/schema/atoms/deliveryMethod.schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+export enum DeliveryMethod {
+  'Virtual' = 'virtual',
+  'In-person' = 'in-person'
```

Comment:
same here for enums best praces would be to have: 
  'Virtual' = 'virtual',
  'InPerson' = 'in-person'